app/container-imp.ts

```diff
@@ -113,7 +114,6 @@ export class AppContainer implements Container {
       lokiUrl: Config.REMOTE_LOGGING_URL,
       lokiLabels: {
         application: getApplicationName().toLowerCase(),
-        job: 'react-native-logs',
```

Comment:
`job` is now specified by the remote logger, if we also specify it here, every job will be the same whether they are incident reports or remote logging sessions. Having separate jobs should allow us to have different lifespans for each job type